src/main/resources/application.yml

```diff
@@ -108,8 +108,10 @@ linkedIn:
     accessTokenUri: https://www.linkedin.com/oauth/v2/accessToken
     userAuthorizationUri: https://www.linkedin.com/oauth/v2/authorization
     clientAuthenticationScheme: query
+    scope: "r_liteprofile r_emailaddress"
   resource:
-    userInfoUri: https://api.linkedin.com/v1/people/~:(email-address,first-name,last-name)?format=json
+    userInfoUri: https://api.linkedin.com/v2/me
+    userEmailUri: https://api.linkedin.com/v2/clientAwareMemberHandles?q=members&projection=(elements*(primary,type,handle~))
```

Comment:
are `userInfoUri` and `userEmailUri` really configuration variables that need to be in our application.yml? we probably just want a `apiHost: https://api.linkedin.com/v2` and then the paths to those URIs can be built in application logic - no person spinning up Ego would want to customize or configure those paths, and would actually break the application if they did.